corehq/util/tests/test_couch.py

```diff
@@ -76,3 +77,121 @@ def test_get_document_or_404_success(self):
         """
         doc = get_document_or_404(MockModel, 'ham', '123')
         self.assertEqual(doc, {'wrapped': {'_id': '123', 'domain': 'ham', 'doc_type': 'MockModel'}})
+
+
+class LoggingDB(object):
```

Comment:
I probably could've used patching to accomplish this same thing, but this seemed easier.